src/OoasGui/OoasGui.csproj

```diff
@@ -13,7 +13,7 @@
     <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
-    <PlatformTarget>AnyCPU</PlatformTarget>
+    <PlatformTarget>x64</PlatformTarget>
```

Comment:
Is this change anticipated?

Reply:
@irvinesunday Please ignore this PR. I am using this PR to test the CI (PR triger)